leases/plugin/local.go

```diff
@@ -23,15 +23,15 @@ import (
 	"github.com/containerd/containerd/leases"
 	"github.com/containerd/containerd/metadata"
 	"github.com/containerd/containerd/plugin"
-	"github.com/containerd/containerd/services"
 )
 
 func init() {
 	plugin.Register(&plugin.Registration{
-		Type: plugin.ServicePlugin,
-		ID:   services.LeasesService,
+		Type: plugin.LeasesPlugin,
+		ID:   "manager",
```

Comment:
Should this be a constant?

Reply:
We don't always use a constant for the ID when there is usually just a single implementation. It does seem like we should use const though since using `GetByID`. I'm not sure it is worth pulling in an extra package though to share the const.

Reply:
I'm fine with it either way; the use of `GetByID` was really why I asked but I agree about the limited value of adding another package here.

Reply:
yeah, if we can find a more consistent way to do it across all the packages, that would be good. `GetByID` is a newer function as well so wasn't used originally in a lot of places, many times it just uses the first implementation. That really isn't great approach though since it doesn't help actually choose amongst multiple plugins should different implementations get loaded... Something we could clean up. Since plugin load is done once though, const vs just copying a string doesn't make much difference really, plugin load will fail every time if it doesn't match.